### DISCUSSION TITLE
Can't connect to RPi Pico

### DISCUSSION THREAD

**Discussion by cheddy51** [micropython/micropython#9998] [Hardware & Peripherals]
I am trying to connect to an RPi Pico from an RPi 3B+ running Raspian Bullseye. The Pico shows up in the File Manager when I connect with the BOOTSEL button pressed. When I try to install the MicroPython interpreter via the Thonny Interpreter Configuration dialogue it appears to work. I get the 'Done!' message and looking at the dialogue that appears when you click Done I see the following:

    Waiting for the port
    Found 2e8a:0005 at /dev/ttyACM0
    Done!

That seems to indicate that the firmware install worked, but I don't see any REPL prompt in Thonny and I get:

    Backend terminated or disconnected. Use 'Stop/Restart' to restart.

But 'Stop/Restart' doesn't do anything, nor do any of the other Thonny commands.

I have tried to flash both the 'MicroPython_RenameMainDotPy.uf2' file and the 'flash_nuke.uf2' file, but neither makes any difference, and I typically get an error saying 'An operation is already pending' and I haven't found any way to clear this seemingly blocked operation.

I have tried two Picos, with the same results for both. Both Picos were working fine a few months ago, and the only thing I have done in the meantime is an 'apt update' followed by an 'apt upgrade'. Apt tells me that Thonny is the latest version when I try to upgrade it.

I also thought it might be a bad USB cable, but I have tried 3 or 4 different cables with exactly the same results.

This is extremely frustrating. Does anyone know what the problem might be?

**Comment by robert-hh** [comment]
Does it work with sudo? If yes , xou have to add your user name to the dialout group.
